django/db/models/query.py

```diff
@@ -559,13 +559,24 @@ def in_bulk(self, id_list=None):
         """
         Return a dictionary mapping each of the given IDs to the object with
         that ID. If `id_list` isn't provided, evaluate the entire QuerySet.
+
+        Some databases (e.g. SQLite) have a limit on the number of query
```

Comment:
I think this would be more appropriate as an inline comment about the `if batch_size ...` line. `# If the database has a limit on the number of query parameters (e.g. SQLite), retrieve objects in batches if necessary.`